src/test-project/src/test/java/com/redhat/devtools/intellij/commonUiTestLibrary/fixturesTest/mainIdeWindow/toolWindowsPane/buildToolPane/MavenBuildToolPaneTest.java

```diff
@@ -0,0 +1,77 @@
+ ******************************************************************************/
+package com.redhat.devtools.intellij.commonUiTestLibrary.fixturesTest.mainIdeWindow.toolWindowsPane.buildToolPane;
+
+import com.intellij.remoterobot.fixtures.JTreeFixture;
+import com.redhat.devtools.intellij.commonUiTestLibrary.LibraryTestBase;
+import com.redhat.devtools.intellij.commonUiTestLibrary.fixtures.mainIdeWindow.toolWindowsPane.BuildView;
+import com.redhat.devtools.intellij.commonUiTestLibrary.fixtures.mainIdeWindow.toolWindowsPane.ToolWindowsPane;
+import com.redhat.devtools.intellij.commonUiTestLibrary.fixtures.mainIdeWindow.toolWindowsPane.buildToolPane.MavenBuildToolPane;
+import com.redhat.devtools.intellij.commonUiTestLibrary.utils.project.CreateCloseUtils;
+import com.redhat.devtools.intellij.commonUiTestLibrary.utils.testExtension.ScreenshotAfterTestFailExtension;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+import java.time.Duration;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+/**
+ * Maven Build Tool Pane test
+ *
+ * @author zcervink@redhat.com
+ */
+@ExtendWith(ScreenshotAfterTestFailExtension.class)
+class MavenBuildToolPaneTest extends LibraryTestBase {
+    private static final String projectName = "maven_build_tool_pane_java_project";
+    private static ToolWindowsPane toolWindowsPane;
+    private static MavenBuildToolPane mavenBuildToolPane;
+
+    @BeforeAll
+    public static void prepareProject() {
+        CreateCloseUtils.createNewProject(remoteRobot, projectName, CreateCloseUtils.NewProjectType.MAVEN);
+        toolWindowsPane = remoteRobot.find(ToolWindowsPane.class, Duration.ofSeconds(10));
+        toolWindowsPane.openMavenBuildToolPane();
+        mavenBuildToolPane = toolWindowsPane.find(MavenBuildToolPane.class, Duration.ofSeconds(10));
+    }
+
+    @AfterAll
+    public static void closeCurrentProject() {
+        CreateCloseUtils.closeProject(remoteRobot);
+    }
+
+    @Test
+    public void buildProjectTest() {
+        mavenBuildToolPane.buildProject("install");
+        boolean isBuildSuccessful = toolWindowsPane.find(BuildView.class, Duration.ofSeconds(10)).isBuildSuccessful();
+        assertTrue(isBuildSuccessful, "The build should be successful but is not.");
+    }
+
+    @Test
+    public void reloadAllMavenProjectsTest() {
+        mavenBuildToolPane.reloadAllMavenProjects();
+    }
+
+    @Test
+    public void collapseAllTest() {
+        mavenTargetTree().expandAll();
+        int itemsCountBeforeCollapsing = mavenTargetTree().collectRows().size();
+        mavenBuildToolPane.collapseAll();
+        int itemsCountAfterCollapsing = mavenTargetTree().collectRows().size();
+        assertTrue(itemsCountAfterCollapsing < itemsCountBeforeCollapsing, "The 'Collapse All' operation was unsuccessful.");
+    }
+
+    private JTreeFixture mavenTargetTree() {
```

Comment:
again, it is defined in proper class. no need for this use here.

Reply:
Fixed.